core/RhythmStubsSimple/src/main/java/org/jjazzlab/rhythmstubssimple/RhythmStubSimpleProviderImpl.java

```diff
@@ -0,0 +1,151 @@
+/*
+ *  DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ * 
+ *  Copyright @2025 Jerome Lelasseux. All rights reserved.
+ *
+ *  This file is part of the JJazzLab software.
+ *   
+ *  JJazzLab is free software: you can redistribute it and/or modify
+ *  it under the terms of the Lesser GNU General Public License (LGPLv3) 
+ *  as published by the Free Software Foundation, either version 3 of the License, 
+ *  or (at your option) any later version.
+ *
+ *  JJazzLab is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ * 
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with JJazzLab.  If not, see <https://www.gnu.org/licenses/>
+ * 
+ *  Contributor(s): 
+ */
+package org.jjazzlab.rhythmstubssimple;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.jjazz.harmony.api.TimeSignature;
+import org.jjazz.rhythm.api.Division;
+import org.jjazz.rhythm.api.Genre;
+import org.jjazz.rhythm.api.Rhythm;
+import org.jjazz.rhythm.api.RhythmFeatures;
+import org.jjazz.rhythm.api.TempoRange;
+import org.jjazz.rhythm.spi.RhythmProvider;
+import org.jjazz.rhythm.spi.StubRhythmProvider;
+import org.jjazz.utilities.api.MultipleErrorsReport;
+import org.jjazz.utilities.api.ResUtil;
+import org.openide.util.lookup.ServiceProvider;
+import org.openide.util.lookup.ServiceProviders;
+
+/**
+ * 
+ */
+@ServiceProviders(value =
+{
+    @ServiceProvider(service = StubRhythmProvider.class),
+    @ServiceProvider(service = RhythmProvider.class)            // So that it appears in the Rhythm selection dialog box
+})
+public  class RhythmStubSimpleProviderImpl implements StubRhythmProvider
+{
+
+    public static final String ID = "StubRhythmProviderID";
+    private Info info;
+    private ArrayList<Rhythm> rhythms = new ArrayList<>();
+
+    public RhythmStubSimpleProviderImpl()
+    {
+        info = new Info(ID, ResUtil.getString(getClass(), "DUMMY_RHYTHMS"), ResUtil.getString(getClass(), "DUMMY_RHYTHMS_DESC"), "JL", "1.0");
```

Comment:
Don't bother with internationalization for a test mock (with the crowdin integration, this will get translated by users for nothing...).